migration/identity-migration/src/test/java/io/camunda/migration/identity/RoleMigrationHandlerTest.java

```diff
@@ -8,201 +8,83 @@
-    //            .collect(
-    //                Collectors.toMap(permissionType -> permissionType, permissionType ->
-    // Set.of("*")));
-    //    Arrays.asList(
-    //            AuthorizationResourceType.PROCESS_DEFINITION,
-    //            AuthorizationResourceType.DECISION_DEFINITION,
-    //            AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION)
-    //        .forEach(
-    //            resourceType ->
-    //                assertThat(authorizations)
-    //                    .contains(
-    //                        new PatchAuthorizationRequest(
-    //                            1, PermissionAction.ADD, resourceType, defaultPermissionMap)));
-    //    assertThat(authorizations)
-    //        .contains(
-    //            new PatchAuthorizationRequest(
-    //                1,
-    //                PermissionAction.ADD,
-    //                AuthorizationResourceType.APPLICATION,
-    //                Map.of(PermissionType.ACCESS, Set.of("operate"))));
-    //    assertThat(authorizations)
-    //        .contains(
-    //            new PatchAuthorizationRequest(
-    //                2,
-    //                PermissionAction.ADD,
-    //                AuthorizationResourceType.APPLICATION,
-    //                Map.of(PermissionType.ACCESS, Set.of("tasklist", "operate"))));
+    final var rolesResult = ArgumentCaptor.forClass(CreateRoleRequest.class);
+    verify(roleServices, times(4)).createRole(rolesResult.capture());
+    final List<CreateRoleRequest> requests = rolesResult.getAllValues();
+    assertThat(requests).hasSize(4);
+    assertThat(requests.getFirst().roleId()).isEqualTo("developer");
+    assertThat(requests.getFirst().name()).isEqualTo("Developer");
+    assertThat(requests.get(1).roleId()).isEqualTo("operationsEngineer");
+    assertThat(requests.get(1).name()).isEqualTo("Operations Engineer");
+    assertThat(requests.get(2).roleId()).isEqualTo("taskUser");
+    assertThat(requests.get(2).name()).isEqualTo("Task User");
+    assertThat(requests.get(3).roleId()).isEqualTo("visitor");
+    assertThat(requests.get(3).name()).isEqualTo("Visitor");
   }
 
   @Test
-  void setErrorWhenRoleAlreadyExists() {
-    // given
-
-    when(roleServices.createRole(any()))
-        .thenReturn(
+  public void shouldContinueMigrationIfOnRoleAlreadyExists() {
+    doReturn(CompletableFuture.completedFuture(null))
+        .doReturn(
             CompletableFuture.failedFuture(
                 new BrokerRejectionException(
                     new BrokerRejection(
-                        RoleIntent.CREATE,
+                        GroupIntent.CREATE,
                         -1,
                         RejectionType.ALREADY_EXISTS,
-                        "role already exists"))));
-    givenRoles();
+                        "role already exists"))))
+        .doReturn(CompletableFuture.completedFuture(null))
+        .doReturn(CompletableFuture.completedFuture(null))
```

Comment:
🔧  duplicated? or is that needed?
```suggestion
        .doReturn(CompletableFuture.completedFuture(null))
```

Reply:
It's needed